Source/WebKit/UIProcess/API/wpe/qt6/WPEQtView.cpp

```diff
@@ -84,7 +84,7 @@ void WPEQtView::geometryChange(const QRectF& newGeometry, const QRectF&)
         return;
 
     auto* wpeView = webkit_web_view_get_wpe_view(d->m_webView.get());
-    wpe_view_resize(wpeView, d->m_size.width(), d->m_size.height());
+    wpe_view_resized(wpeView, d->m_size.width(), d->m_size.height());
```

Comment:
That is not correct, this only tells the view that it has been resized, but there is no underlying call to e.g. resize the Wayland surfaces. Nowadays resizes need to be handled via `wpe_toplevel_resize`. You can query the WPEView for it's WPEToplevel, and issue the resize request there.

@carlosgcampos - please correct me if I'm wrong.

Reply:
yes, we plan to use  Qt6 qt-wpe-mini-browser.